src/lib/core/error/error-options.ts

```diff
@@ -7,27 +7,24 @@
  */
 
 import {InjectionToken} from '@angular/core';
-import {FormControl, FormGroupDirective, NgForm} from '@angular/forms';
+import {FormGroupDirective, NgForm, NgControl} from '@angular/forms';
 
 /** Injection token that can be used to specify the global error options. */
 export const MD_ERROR_GLOBAL_OPTIONS = new InjectionToken<ErrorOptions>('md-error-global-options');
 
 export type ErrorStateMatcher =
```

Comment:
Ok sounds good, I do want to keep the interface though so that components like the stepper can implement it.

```ts
interface ErrorOptions { ... }
class DefaultErrorOptions implements ErrorOptions { ... }
```